lib/Manager.php

```diff
@@ -1084,7 +1084,7 @@ public function getChangelogRoom(string $userId): Room {
 	 * @param string $objectId
 	 * @return Room
 	 */
-	public function createRoom(int $type, string $name = '', string $objectType = '', string $objectId = ''): Room {
+	public function createRoom(int $type, string $name = '', string $objectType = '', string $objectId = '', string $password = ''): Room {
```

Comment:
password is not set during the insert.

Also it needs to be hashed before saving:
https://github.com/nextcloud/spreed/blob/c7e07968fe56e1f0bebaef3bb58db68c2eb1d42f/lib/Service/RoomService.php#L744